column/api/controller/credential_controller.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from flask import request
+import flask_restful
+from six.moves import http_client
+
+from column.api import manager
+
+LOG = logging.getLogger(__name__)
+
+
+class Credential(flask_restful.Resource):
```

Comment:
I don't think the naming of the class and methods make much sense anymore. We're not "getting" credentials.  We're getting an encrypted or decrypted form of a string.